### DISCUSSION TITLE
webgui does not accept connections 10.3.141.3

### DISCUSSION THREAD

**Discussion by riker65** [RaspAP/raspap-webgui#1192] [Q&A] [answered]
I can ssh into standard 10.3.141.1 but website is not allowing connection
port not open

any hints

**Comment by billz** [comment]
If this is not a clean install of RaspAP, you may have other services using port 80. Check this with:

```
sudo netstat -tulpn | grep 80
```

**Comment by riker65** [comment]
thanks nothing is blocking, but looks like service http is not running

how to verify?

if install is not clean, how to do I clean up the mess I made?

thanks Thomas

**Comment by riker65** [comment]
maybe guide me to an overview how it should look like

which services should run?

thanks

**Comment by riker65** [comment]
Hi now
seeing the message on rpi
http://localhost

443 forbidden

**Comment by riker65** [comment]
checked the permissions in /var/www
look ok 

followed

[https://docs.raspap.com/manual/#create-the-web-application](url)

**Comment by billz** [comment]
I strongly suggest you start with a clean SD and use this project's Quick Installer or follow the manual setup steps.

**Comment by riker65** [comment, marked as answer]
solved by new clean install 

thanks
